README.md

```diff
@@ -9,12 +9,17 @@ Recommended practices
 ===
 
 - Queries
-  - Should be defined in files named as `sql/table_version.sql` e.g. `sql/clients_daily_v6.sql`
+  - Should be defined in files named as `sql/table_version.sql` e.g.
+    `sql/clients_daily_v6.sql`
   - Should not specify a project or dataset in table names to simplify testing
   - Should be [incremental](#incremental-queries)
   - Should filter input tables on partition and clustering columns
   - Should use `_` prefix in generated column names not meant for output
   - Should not use jinja templating on the query file in Airflow
+  - Should use `days_since_seen_` in column names to mean "any aggregated
```

Comment:
```suggestion
  - Should use `days_since_seen_` in daily aggregation column names to mean "any aggregated
```

This "should" is specific to aggregations per entity per day, and I think that needs to be made more explicit here.

Reply:
I'm removing this because I think it's too complicated and specific to fit well here, and also I think it's more of a judgement call supported by prior art.